requirements.txt

```diff
@@ -37,7 +36,7 @@ Babel==2.14.0
 citeproc-py==0.6.0
 boto3==1.34.60
 django-waffle==4.1.0
-pymongo==4.6.3 # install to get bson module
+pymongo[ocsp]==3.13.0 # install to get bson module
```

Comment:
I am not familiar with `pymongo`, any details and/or references on this change?

Reply:
This change probably has been lost in some merge, but it fixes the problem of incompatibility between mongodb 3.4 and python 3.10+